### PR TITLE
removing repetitive content in park_views and photo_views

### PR DESCRIPTION
**Pull request by shaibird**
**Description**
Implemented sql helper functions into parks and photos by  removing sql connection code from each module and inserting the get_all() and get_single() helper functions where appropriate. get_all and get_single were imported from sql_helper.py.
Added a get_photo_by_user_id function

**Steps to test get_all**
Fetch the sc-updating_get branch and switch to it
Start/restart debugger
Send a GET request for all parks data (e.g. http://localhost:8088/parks)
Verify that 200 status code and the entire list of dictionaries are received
Repeat process for the following GET requests:
http://localhost:8088/photos

**Steps to test get_single**
Fetch the sc-updating_get branch and switch to it
Start/restart debugger
Send a GET request for an existing park(e.g. http://localhost:8088/parks/1)
Verify that 200 status code and the single dictionary with id = 1 are received
Repeat process for the following GET requests:
http://localhost:8088/photos/1

**Steps to test get_photo_by_user_id**
Fetch the sc-updating_get branch and switch to it
Start/restart debugger
Send a GET request for existing photos with user_id = 1 (http://localhost:8088/photos?user_id=1)
Verify that 200 status code and the single dictionary with id = 1 are received

### CODE REVIEW

File: views/photo_views.py [hunk 1]
```diff
@@ -1,54 +1,81 @@
...
+    Returns:
+        list: of all photos associated with the user_id foreign key
+    """
+
+    sql=("""
+       SELECT
+            p.id,
+            p.url,
+            p.user_id,
+            p.park_id,
```
Comment (JohnMDoll) [inline views/photo_views.py]: extra comma at end of line 69

File: views/photo_views.py [hunk 2]
```diff
@@ -1,54 +1,81 @@
...
+        dict: photo dictionary
+    """
 
-        db_cursor.execute("""
+    sql = """
         SELECT
             p.id,
             p.url,
             p.user_id,
             p.park_id,
```
Comment (JohnMDoll) [inline views/photo_views.py]: extra comma at end of line 43

File: views/park_views.py [hunk 3]
```diff
@@ -17,25 +15,21 @@ def get_all_parks():
             p.latitude,
             p.longitude,
         FROM parks p
```
Comment (JohnMDoll) [inline views/park_views.py:17]: extra comma at end of line 16

File: views/park_views.py [hunk 4]
```diff
@@ -46,14 +40,10 @@ def get_single_park(id):
...
```
Comment (JohnMDoll) [inline views/park_views.py:41]: extra comma at end of line 40

File: views/photo_views.py [hunk 5]
```diff
@@ -1,54 +1,81 @@
...
+        list: All photo dictionaries"""
 
-        db_cursor.execute("""
+    sql = """
         SELECT
             p.id,
             p.url,
             p.user_id,
             p.park_id,
         FROM photos p
```
Comment (JohnMDoll) [inline views/photo_views.py:15]: extra comma at end of line 14

**Review by vanessaspear — APPROVED** [review]
Looks good after John's changes are implemented!

**Review by JohnMDoll — APPROVED** [review]

**Pull request merged**